web/wp-content/themes/bumblebee/functions.php

```diff
@@ -444,3 +444,34 @@ function enable_svg_upload( $existing_mimes = array() ) {
 	return $existing_mimes;
 }
 add_filter( 'upload_mimes', 'enable_svg_upload' );
+
+/**
+ * Add In Content NL Module AFTER the_content
+ *
+ * @param  content $content Append NL to the_content
+ */
+function newsletter_after_the_content( $content ) {
```

Comment:
This function seems to be unused, do we need to keep it?

Reply:
Created ticket https://readersdigest.atlassian.net/browse/DRDT-118 to address such issues